snaps_tests/__init__.py

```diff
@@ -193,14 +193,15 @@ def install_store_snap(self, snap_name):
     def install_snap(self, snap_local_path, snap_name, version,
                      devmode=False):
         if not config.get('skip-install', False):
-            self.snappy_testbed.copy_file(snap_local_path, '/home/ubuntu')
-            snap_file_name = os.path.basename(snap_local_path)
-            snap_path_in_testbed = os.path.join(
-                '/home/ubuntu/', snap_file_name)
-            # Remove the snap file from the testbed.
+            tmp_in_testbed = self.snappy_testbed.run_command(
+                'mktemp -d').strip()
```

Comment:
well, potentially another machine. We are not running in a remote testbed anymore, so it would be worth thinking if it's better to simplify these scripts.
However, if I understand correctly, the snapcraft snap will only work on classic. Then we still need to test the snaps in remote core some times.